www/templates/main.html

```diff
@@ -26,10 +28,12 @@
     <ion-nav-view name="main-metrics"></ion-nav-view>
   </ion-tab>
 
-  <!-- Heatmap Tab -->
-  <ion-tab title="{{'main-heatmap.title' | translate}}" icon="ion-fireball" href="#/root/main/heatmap">
+  <!-- Heatmap Tab 
+  <ion-tab title="Heatmap" icon="ion-fireball" href="#/root/main/heatmap">
```

Comment:
You are only commenting out the heatmap and game tabs here, but are retaining the dashboard. Is this what you want?

Reply:
Yeah